modules/rostests/kmtests/ntos_cc/CcCopyRead_drv.c

```diff
@@ -315,17 +313,23 @@ TestIrpHandler(
         Status = STATUS_SUCCESS;
     }
 
+    // CcCopyRead never sets STATUS_PENDING, especially with wait=TRUE.
+    ASSERT(Status != STATUS_PENDING);
+
+// NB: Same block in IoReadWrite_drv.c, MmMapLockedPagesSpecifyCache_drv.c and NtCreateSection_drv.c.
+// Nonetheless, keep the following STATUS_PENDING checks, as a template.
     if (Status == STATUS_PENDING)
     {
-        IoMarkIrpPending(Irp);
-        IoCompleteRequest(Irp, IO_NO_INCREMENT);
-        Status = STATUS_PENDING;
+//        IoMarkIrpPending(Irp);
+// Remove code added as is in r71445.
```

Comment:
What's the purpose of such removal?

Reply:
`Status` is obviously redundant. `IoCompleteRequest` is wrong, unless I misunderstood. `IoMarkIrpPending` is wrong too, isn't it?